### ISSUE TITLE
Allow subsetting TreeOfLifeClassifier

### ISSUE THREAD

**Issue by johnbradley** [Imageomics/pybioclip#56]
Provide a way to predict using a subset of the TreeOfLifeClassifier embeddings.
This came out of #54.

**Comment by johnbradley** [comment]
Each embedding is associated with a set of values: kingdom, phylum, class, order, family, genus, species_epithet, species, and common_name. Example:
```
kingdom,phylum,class,order,family,genus,species_epithet,species,common_name
Animalia,Chordata,Mammalia,Carnivora,Ursidae,Ursus,arctos,Ursus arctos,Kodiak bear
```


To use a subset of the TOL embeddings from the command line I was thinking a user could supply a CSV using the column names.
For example to subset for the orders within mammals create a CSV with contents:
```
class
Rodentia
Chiroptera
Eulipotyphla
Primates
Artiodactyla
Carnivora
...
```
Within python this could be a DataFrame with the particular columns.

**Comment by johnbradley** [comment]
Fixed by #59 and #63
